plugins/modules/monitoring/datadog/datadog_event.py

```diff
@@ -131,8 +150,10 @@ def main():
 
     options = {
         'api_key': module.params['api_key'],
-        'app_key': module.params['app_key']
+        'app_key': module.params['app_key'],
     }
+    if module.params['api_key'] is not None:
```

Comment:
`api_key` and `app_key` are required, and module will throw an error when they're `None`, why check here again if it `is not None`? Did you mean this?
```suggestion
    if module.params['api_host'] is not None:
```

Reply:
```suggestion
    if module.params['api_host']:
```

Reply:
You're absolutely right! @aminvakil 
Committing...

Reply:
@russoz I think I'd rather keep the code a bit descriptive

Reply:
Okie, this boils down to personal preferences. :-)